src/index.html

```diff
@@ -10,7 +10,34 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form 
+      class="form_field"
+      action="https://www.google.com/" 
+      method="get" 
+      data-qa="big"
+    >
+      <label>
+        <input
+          class="search_bar1 search_bar"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <form 
+      class="form_field"
+      action="https://www.google.com/" 
+      method="get" 
+      data-qa="small"
+    >
+      <label>
+        <input
+          class="small_search_bar  small_search_bar1"
```

Comment:
You have a lot of repeatable styles. 
So you need to refactor you code. I will give you an EXAMPLE, how to do it

Following the BEM-methodology - use the `block` selector for the FIRST input (`class="search-bar"`), where you apply common input styles

```
.search-bar {
  {{common_input_styles}}
}
```

And then you apply `modifier` to the SECOND input (`class="search-bar search-bat--small"`) that have ONLY differences with base class search-bar
```
.search-bar--small {
  {{only_those_styles_that_have_difference}}
}
```
So in this way you follow DRY (Don't repeat yourself) principe
Apply this principe all over the styles

```suggestion
          class="search-bar search-bar--small"
```